cadence/scripts/get_nft_collections_for_nft_type.cdc

```diff
@@ -1,5 +1,5 @@
-import NFTCatalog from "../contracts/NFTCatalog.cdc"
+import NFTCatalog from "NFTCatalog"
```

Comment:
I'm seeing in some transactions you use the `import X from "X"` format for the imports, but in others you use `import "x"`. Probably would be good to standardize